wsw/preprocessing.py

```diff
@@ -0,0 +1,92 @@
+# Project root directory
+from definitions import ROOT_DIR
+
+# File and I/O
+import warnings
+import os
+
+# Audio I/O and processing
+import librosa
+import soundfile as sf
+
+# Multi-processing tools
+from multiprocessing import Pool
+import matplotlib.pyplot as plt
+
+
+# Load an audio file in librosa from file path
+def read_audio(file_path, sr=22050):
```

Comment:
For each function you write, it is customary to provide docstrings on what the expected inputs and outputs are and the function's behaviour.  

Any format from https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format is acceptable.

Reply:
Perfect, thanks. I added docstrings to all the modules, including the tests.